lib/Components/Divider.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class DoubleDivider extends StatelessWidget{
+
+  @override 
+  Widget build(BuildContext context){
+    final instance = ScreenUtil.getInstance();
+    return Row(
+      children: <Widget>[
+        Expanded(
+            child: Divider(
+              color: Colors.grey.shade100,
```

Comment:
I would adjust the colour of the divider to be shade 300 as the text is inheriting the subtitle theme, which has a shade of grey[350], just to keep it consistent.

Reply:
I would also specify a thickness of 1px on the dividers

Reply:
after experimenting it seems that 350 fits better the trello prototype for the length of the dividers